interface/Cargo.toml

```
@@ -0,0 +1,59 @@
+[package]
+name = "solana-stake-interface"
+version = "0.1.0"
+description = "Instructions and constructors for the Stake program"
+repository = "https://github.com/solana-program/stake"
+edition = "2021"
+readme = "README.md"
+license-file = "../LICENSE"
+
+[dependencies]
+borsh = { version = "1.5.1", features = ["derive", "unstable__schema"], optional = true }
+borsh0-10 = { package = "borsh", version = "0.10.3", optional = true }
+num-traits = "0.2"
+serde = { version = "1.0.210", features = ["derive"], optional = true }
+serde_derive = { version = "1.0.210", optional = true }
```

You don't need the `derive` feature if you're bringing in `serde_derive`
```suggestion
serde = { version = "1.0.210", optional = true }
serde_derive = { version = "1.0.210", optional = true }
```